src/Microsoft.AspNet.Authentication/DataHandler/Serializer/TicketSerializer.cs

```diff
@@ -41,21 +41,29 @@ public static void Write([NotNull] BinaryWriter writer, [NotNull] Authentication
             writer.Write(FormatVersion);
```

Comment:
I'm not sure: the serialization format is not really changed (this PR only gives `-1` a special meaning).